src/components/structured-text-block/structured-text-block.vue

```diff
@@ -175,8 +190,6 @@
   .structured-text {
     grid-column: var(--grid-content);
     word-wrap: break-word;
-    display: flex;
-    flex-direction: column;
```

Comment:
The flex was used to center the buttons-list. I've used auto margins to solve that, so we could get rid of floats and make the good old floats work, as these work easily context independently.